subprojects/packagefiles/theora/lib/meson.build

```diff
@@ -0,0 +1,204 @@
+encoder_sources = files(
+    'apiwrapper.c',
+    'bitpack.c',
+    'dequant.c',
+    'fragment.c',
+    'idct.c',
+    'internal.c',
+    'state.c',
+    'quant.c',
+    'analyze.c',
+    'fdct.c',
+    'encfrag.c',
+    'encapiwrapper.c',
+    'encinfo.c',
+    'info.c', # not in SConstruct or Makefile.am
+    'encode.c',
+    'enquant.c',
+    'huffenc.c',
+    'mathops.c',
+    'mcenc.c',
+    'rate.c',
+    'tokenize.c',
+)
+
+decoder_sources = files(
+    'apiwrapper.c',
+    'bitpack.c',
+    'decapiwrapper.c',
+    'decinfo.c',
+    'decode.c',
+    'dequant.c',
+    'fragment.c',
+    'huffdec.c',
+    'idct.c',
+    'info.c',
+    'internal.c',
+    'quant.c',
+    'state.c',
+)
+
+if asm
+    if conf.get('CPU_x86_64', false)
+        if cc.get_id() != 'msvc'
```

Comment:
here and elsewhere, I think it makes more sense to use get_argument_syntax instead of get_id.

Reply:
It's the incorrect semantics. `get_argument_syntax` covers Clang/CL, while the one that can't do inline assembly is MSVC itself.

Reply:
ah OK.